packages/commonwealth/client/scripts/views/components/component_kit/showcase_helpers.tsx

```diff
@@ -0,0 +1,11 @@
+export const optionList = [
+  { value: '1inch', label: '1 Inch' },
+  { value: 'atom-blockchain', label: 'Atom Blockchain' },
+  { value: 'dydx', label: 'dydx' },
+  { value: 'edgeware', label: 'Edgeware' },
+  { value: 'maia-dao', label: 'Maia Dao' },
+  { value: 'oraichain', label: 'Oraichain' },
+  { value: 'stargate-finance', label: 'Stargate Finance' },
+  { value: 'stargaze', label: 'Stargaze' },
+  { value: 'terra-classic', label: 'Terra Classic' },
+]
```

Comment:
I can see that your prettier issue is still not resolved

Reply:
yes you are right, but I am not talking about single quotes here. I know this is not properly formatted file, because there is missing semicolon at the and of the array.

Reply:
But this has been fixed in the meanwhile, thanks.